lib/content-services/src/lib/dialogs/dialog/dialog.model.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum DialogSize {
```

Comment:
it is better to avoid enums, https://alfresco.atlassian.net/wiki/spaces/TECH/pages/1646559233/FE+code+best+practices

Reply:
oh, I don't have access to this page, I'll create ticket

Reply:
done, thanks